src/Microsoft.DotNet.Web.ProjectTemplates/EmptyWeb-FSharp.fsproj.in

```diff
@@ -15,9 +15,6 @@
   <ItemGroup Condition="'$(TargetFrameworkOverride)' == ''">
     <PackageReference Include="Microsoft.AspNetCore.All" Version="${MicrosoftAspNetCoreAllPackageVersion}" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFrameworkOverride)' != ''">
```

Comment:
Looks like an earlier bad merge. The package reference re-appears right after this entry.


cc @jbagga